frontend/public/components/image-stream.tsx

```diff
@@ -208,6 +236,11 @@ type ImageStreamTagsRowProps = {
   statusTag: any,
 };
 
+export type ImageStreamManipulationHelpProps = {
+  registryAddress: string,
+  isRegistryPublic: boolean,
```

Comment:
The type declarations should end with `;` instead of `,`. (I know it's wrong elsewhere in the file.)